src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -33,6 +36,39 @@
     'Unsigned-integer-overflow',
 ]
 
+MEMORY_TOOLS_LABELS = [
+    {
+        'token': 'AddressSanitizer',
+        'label': 'Memory-AddressSanitizer'
+    },
+    {
+        'token': 'LeakSanitizer',
+        'label': 'Memory-LeakSanitizer'
+    },
+    {
+        'token': 'MemorySanitizer',
+        'label': 'Memory-MemorySanitizer'
+    },
+    {
+        'token': 'ThreadSanitizer',
+        'label': 'ThreadSanitizer'
+    },
+    {
+        'token': 'UndefinedBehaviorSanitizer',
+        'label': 'UndefinedBehaviorSanitizer'
+    },
+    {
+        'token': 'afl',
+        'label': 'AFL'
+    },
+    {
+        'token': 'libfuzzer',
+        'label': 'LibFuzzer'
+    },
+]
+
+STACKFRAME_LINE_REGEX = re.compile(r'\s*#\d+\s+0x[0-9A-Fa-f]+\s*')
```

Comment:
This is somewhat similar to the regex we have in stack_analyzer: https://github.com/google/clusterfuzz/blob/b5852e8885a80572e7950400649753d270c1cbb1/src/python/crash_analysis/stack_parsing/stack_analyzer.py#L529

Would it make sense to unify two of these?

Reply:
They look like they're matching different things. The one in stack_analyzer expects a '+' literal before the hex number (for removing offsets).